plugins/telemetry-otel/src/main/java/org/opensearch/telemetry/OTelTelemetrySettings.java

```diff
@@ -110,4 +112,37 @@ private OTelTelemetrySettings() {}
         Setting.Property.NodeScope,
         Setting.Property.Final
     );
+
+    /**
+     * Max scale setting for the {@link io.opentelemetry.sdk.metrics.internal.view.Base2ExponentialHistogramAggregation}
+     */
+    public static final Setting<Integer> OTEL_METRICS_HISTOGRAM_EXPONENTIAL_MAX_SCALE = Setting.intSetting(
```

Comment:
@Gaganjuneja I believe none of the metric should be controlled over global settings (like this ones):
 - changing these settings will impact **everyone**, in unpredictable (for consumers) ways 
 - the consumers (plugins, extensions, even core) have no control over these settings whatsoever

At creation time, we should allow consumers to configure the metrics the way the want to, and by no means we should alter that implicitly.